src/icons/RushingClock.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
will use the one from #5618 instead once it is merged

Reply:
i think you can keep this since it's already here and this PR will be merged before the next release